unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -390,24 +390,40 @@ void test_spmv(lno_t numRows,size_type nnz, lno_t bandwidth, lno_t row_size_vari
   Kokkos::fill_random(input_xt,rand_pool,randomUpperBound<ScalarX>(10));
   Kokkos::fill_random(output_yt,rand_pool,randomUpperBound<ScalarY>(10));
 
-  std::vector<char> nonTransModes = {'N', 'C'};
-  std::vector<char> transModes = {'T', 'H'};
+  std::vector<char> nonTransModes = {'N'};
+  std::vector<char> transModes = {'T'};
+  std::vector<double> testAlphaBeta = {0.0, 1.0};
+  if(heavy)
+  {
+    nonTransModes.push_back('C');
+    transModes.push_back('H');
+    testAlphaBeta.push_back(-1.0);
+    testAlphaBeta.push_back(2.5);
```

Comment:
Is this just to reduce unit test execution time?

Reply:
Correct, I only enabled 'heavy' testing on much smaller matrices, since the time to run all these cases adds up.